batcher/batcher/batcher.go

```diff
@@ -19,14 +20,22 @@ type lastBatchData struct {
 	txHash string
 }
 
+type validatorSetChange struct {
+	validators *validatorobserver.Validators
```

Comment:
track only
```
type ValidatorsChainData struct {
	Keys       []eth.ValidatorChainData
	SlotNumber uint64
}
```
because each batcher knows on which chain it operates